README.md

```diff
@@ -481,6 +482,13 @@ the crystal system.
 | ivQuadField | val | Minimum field for quadratic IV expression | V/m  |
 | ivQuadPower | exp | Exponent: rate = Rate*(E^2-Field^2)^(exp/2) | none |
 
+The keywords l0_e and l0_h are optional. If they are not specified in
+config.txt, they will be computed from other physical constants: 
+
+l0 = pi*hbar^4*rho/(2*m^3*Dac^2)
```

Comment:
Nice; thank you!  I think for purposes of the README file, it might be easier for users to spell out the parameter names.  Maybe something like
```
l0 = pi*hbar*4*density / (2*mass^3*acDeform^2)
```

Reply:
I'll update that part very soon! ComputeCanada is very very slow and I can't even compile the changes (I've been waiting for 1 hour and reconnecting to the server does not help).

Reply:
Done!